website/docusaurus.config.js

```diff
@@ -119,6 +119,10 @@ function remarkDirectiveBabel8Plugin({ renderBabel8 }) {
 }
 
 const siteConfig = {
+  future: {
+    // See https://docusaurus.io/blog/releases/3.6
+    experimental_faster: true,
```

Comment:
@slorber Thank you for the PR. Will docusaurus 4 remove this option and completely switch to Rspack/SWC/LightningCSS?

Reply:
👍 

I think we'll keep it for v4, but maybe it will become true by default.


It's also possible to enable all the other options except replacing Babel loader by SWC if you really want your site to use Babel. 


Note: if you want to keep Babel, we have a `{webpack: {jsLoader: fn}}` hook to plug your own custom loader. Even if we remove the faster options and make SWC the default, this hook remains an escape hatch use use Babel, OXC, Esbuild or whatever you want.